sdk/entra/Microsoft.Azure.WebJobs.Extensions.AuthenticationEvents/README.md

```diff
@@ -18,102 +18,28 @@ dotnet add package Microsoft.Azure.WebJobs.Extensions.AuthenticationEvents
 
 ### Prerequisites
 
-- **Azure Subscription:** To use Azure services, including Azure Functions, you'll need a subscription. If you do not have an existing Azure account, you may sign up for a [free trial](https://azure.microsoft.com/free/dotnet/) or use your [Visual Studio Subscription](https://visualstudio.microsoft.com/subscriptions/) benefits when you [create an account](https://account.windowsazure.com/Home/Index).
+- **Azure Subscription:** To use Azure services, including Azure Functions, you'll need a subscription. If you do not have an existing Azure account, you may sign up for a [free trial](https://azure.microsoft.com/free/dotnet/) or use your [Visual Studio Subscription](https://visualstudio.microsoft.com/subscriptions/) benefits when you create an account.
 
 ### Authenticate the client
 
-When the Microsoft Entra authentication events service calls your custom extension, it sends an `Authorization` header with a `Bearer {token}`. This token represents a [service to service authentication](https://learn.microsoft.com/azure/active-directory/develop/v2-oauth2-client-creds-grant-flow) in which:
+When the Microsoft Entra authentication events service calls your custom extension, it sends an `Authorization` header with a `Bearer {token}`. This token represents a [service to service authentication](https://learn.microsoft.com/entra/identity-platform/v2-oauth2-client-creds-grant-flow) in which:
```

Comment:
Should we remove this section? It's duplicate info

Reply:
Where is it duplicated?

Reply:
I think that you misunderstand what this section is for.   Think of it in terms of a new developer adding your trigger for the first time.   What configuration do they need to provide the trigger to authenticate? 

For most services, this is a connection string, SAS key, or something that indicates that a credential should be used.   For example, see [Event Hubs extension](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/eventhub/Microsoft.Azure.WebJobs.Extensions.EventHubs/README.md#authenticate-the-client) docs.

Reply:
Sure, it would be tough for a developer to get to this section if they have not already seen the custom extension documentation. 

I can see value in having description of the authentication variables here as it some what decoupled from customextension, and if the developer wanted could set up their own sts service that makes a call to az functions. With that mind, I think we can leave this info here, maybe even update it to match the docs exactly.

Reply:
I like this the way it is. It provides enough information for the developer to get started. If they wanted to find more information on the variables, they can see it in the docs.

Reply:
@hakimms:  Maybe I'm missing something, but I don't see how I could read this doc and use only the README to get an end-to-end "Hello World" scenario running.

Can you help me understand what I'm overlooking?

Reply:
You're correct that the readme doesn't have all the info to get an end to end scenario running. 

We have 2 extremely well detailed documents to get an azure function running and being utilized.

To get a real end-to-end "Hello World" scenario running, you would need to follow the instructions from this [link](https://learn.microsoft.com/entra/identity-platform/custom-extension-tokenissuancestart-setup?tabs=visual-studio%2Cazure-portal&pivots=nuget-library) which is listed below in the `How To Get Started` section and [this doc](https://learn.microsoft.com/en-us/entra/identity-platform/custom-extension-tokenissuancestart-configuration?tabs=azure-portal%2Cworkforce-tenant) which is listed as next steps under the first link. 

The first link are instructions to set up an azure function using this library. This includes everything to create a function, debug it locally and deploy it. The second link are instructions to setup a custom authentication extension app in the entra id/CIAM tenant portal and have that app call the azure function.

Reply:
One of the requirements for an Azure SDK ReadMe file is to have a "Hello World" scenario for developers to get started using the library.   This must be self-contained and not require a developer to click over to different places and look at other documentation.

Reply:
I have updated this doc to develop and deploy the azure function using this library. Please take a look @HarmanDhunna and @jsquire

Reply:
Closing this, please re-open if still active